internal/provider/provider.go

```diff
@@ -193,6 +193,10 @@ func New(version string) func() provider.Provider {
 // test assertions will actually give false positives even if properties have not been updated.
 // @see: https://mezmo.atlassian.net/browse/LOG-18104
 func NullifyPlanFields[M ComponentModel](plan *M, schema resourceSchema.Schema) {
+	// TODO(Martin): Figure out if this is really needed and if so fix panic:
+	// reflect.Set: value of type basetypes.ObjectValue is not assignable to
+	// type *destinations.BlobFileConsolidationModel
```

Comment:
There will have to be a special case for this, obviously since it's the first schema to include a non-TF type.  We need this function to work, though.